code/controllers/subsystems/jobs.dm

```diff
@@ -24,17 +12,24 @@ SUBSYSTEM_DEF(jobs)
 	var/list/positions_by_department = list()
 	var/list/job_icons =               list()
 	var/job_config_file = "config/jobs.txt"
+	var/list/dept_list =			   list()
```

Comment:
What's the purpose of this list? Why not just use `SSdepartments.departments` in all instances?